.github/workflows/xrt_master_2021.1.yml

```diff
@@ -212,6 +219,95 @@ jobs:
           USER: ${{ secrets.USER }}
           github-server-url: ${{ secrets.SERVER_URL }}
 
+      - name: Find and copy packages      
+        run: |      
+          cd ${{ env.PATH_DIR }}/artifact    
```

Comment:
Could you move this entire block into a separate script. Would really help with readability